hoge.txt

```diff
@@ -1,3 +1,6 @@
 ABCDEF
 GHIJKL
-123
\ No newline at end of file
+MNOPRS
+123
+456
+789
```

Comment:
Why alphabet to 6 characters but, number to 3 characters?

Reply:
Oh...
Sorry, revise number to 6 characters too.